.github/workflows/performance.yml

```diff
@@ -66,13 +66,13 @@ jobs:
             - name: Compare performance with base branch
               if: github.event_name == 'push'
               # The base hash used here need to be a commit that is compatible with the current WP version
-              # The current one is 34af5829ac9edb31833167ff6a3b51bea982999c and it needs to be updated every WP major release.
+              # The current one is bd2a881101727b03b0be09382b34841af5a3c03e and it needs to be updated every WP major release.
```

Comment:
how'd we come up with this commit SHA?

> Block Supports: Change prefix in gutenberg_apply_colors_support to wp_ in dynamic blocks (#51989)

Reply:
nevermind I see the answer in the PR description. though I'm still curious: is that WP beta1? GB beta 1?

Reply:
WP 6.3 beta 1. We don’t do beta releases for the plugin.